cog/command.py

```diff
@@ -24,17 +24,12 @@ def load_requestor_(self):
 
     def load_args_(self):
         self.argc_ = 0
+        self.args_ = []
         cog_argc = os.getenv("COG_ARGC")
         if cog_argc is not None:
             self.argc_ = int(cog_argc)
-        self.args_ = []
-        if self.argc_ == 0:
-            return
-        if self.argc_ == 1:
-            self.args_.append(os.getenv(self.index_to_arg_var_(0)))
-        else:
-            for i in range(0, self.argc_):
-                self.args_.append(os.getenv(self.index_to_arg_var_(i)))
+        for i in range(self.argc_):
+            self.args_.append(os.getenv(self.index_to_arg_var_(i)))
```

Comment:
:+1: this is much cleaner than the forest of `if-else` statements I had written.